### PR TITLE
Add `notebook` utility

### PR DESCRIPTION
**Pull request by ReubenFrankel**
### Checklist

- [x] Add/update the file in the appropriate folder (`/taps` or `/targets`). The name of the file should match the name of the tap. If there is already one, add a descriptor to the name such as `-search`.
- [x] Add/update the PNG logo image in `/assets/logos/<taps or targets>`. The image name must match the YAML file name.
- [x] Tag `@tayloramurphy` or `@pnadolny13` to flag it for review. Or post to the [#hub](https://meltano.slack.com/archives/C01UGBSJNG5) channel on Meltano slack.

## Reviewer Checklist

- [ ] Validate file against JSON Schema. https://www.jsonschema.net/home is an option.
- [ ] Build website locally and validate everything works.

### CODE REVIEW

File: _data/meltano/utilities/notebook/matatika.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+description: Jupyter Notebook runner and converter
+name: notebook
+label: Notebook
```
Comment (tayloramurphy) [inline _data/meltano/utilities/notebook/matatika.yml:3]:
```suggestion
label: Jupyter Notebook
```

File: _data/meltano/utilities/notebook/matatika.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  run:
+    args: run $NOTEBOOK_PATH
+    description: Run a notebook.
+  convert:
+    args: convert $NOTEBOOK_PATH -f $NOTEBOOK_FORMAT -c $NOTEBOOK_CONFIG
+    description: Convert a notebook to another format.
+  run-convert:
+    args: run $NOTEBOOK_PATH -f $NOTEBOOK_FORMAT -c $NOTEBOOK_CONFIG
+    description: Run and convert a notebook.
+settings:
```
Comment (pnadolny13) [inline _data/meltano/utilities/notebook/matatika.yml:24]: @ReubenFrankel this looks great!

Are any of these settings required? I don't see any `settings_group_validation`. It seems like you'd need a path in order to do anything. The others look like theyre required depending on the command.

Comment (ReubenFrankel) [reply _data/meltano/utilities/notebook/matatika.yml:24]: Not at any one time, since `install` does not require a path.

Comment (pnadolny13) [reply _data/meltano/utilities/notebook/matatika.yml:24]: @ReubenFrankel ahh ok yeah that makes sense 👍 .

**Review by pnadolny13 — APPROVED** [review]
🚀 

@tayloramurphy this looks good to me, you can merge once youre done reviewing.

**Review by tayloramurphy — APPROVED** [review]

**Comment by ReubenFrankel** [comment]
Related to https://github.com/meltano/hub/issues/191

**Pull request merged**